.travis.yml

```diff
@@ -1,9 +1,24 @@
 language: ruby
 rvm:
+- 2.2
 - 2.1
-- 2.0.0
-- 1.9.3
```

Comment:
Rather than keep it in the matrix and exclude it from all things Jekyll 3, I have dropped it from the matrix and explicitly added one test with Jekyll 2 and Ruby 1.9 on [line 16](https://github.com/jekyll/jekyll-sitemap/pull/83/files#diff-354f30a63fb0907d4ad57269548329e3R16)